aries-site/src/pages/components/tooltip.mdx

```diff
@@ -0,0 +1,67 @@
+import { Example } from '../../layouts';
```

Comment:
We'll need to file a PR on the theme to match the designs, from what I saw we'll need:
```
 tip: {
    content: {
      background: 'background',
      border: {
        color: 'border-weak',
      },
      elevation: 'medium', (NOTE: elevation was getting cut off when I tried this locally, maybe leave as "small" for now but look into Grommet implementation)
      pad: {
        vertical: 'none',
        horizontal: 'small',
      },
      round: 'xsmall',
    },
  },
```

Reply:
Where did you see elevation cut off? I did not see it on my local

Reply:
<img width="378" alt="Screen Shot 2021-02-22 at 9 25 12 AM" src="https://user-images.githubusercontent.com/12522275/108745420-e4d37080-74ef-11eb-90e2-3ac08c709fc7.png">
<img width="633" alt="Screen Shot 2021-02-22 at 9 24 57 AM" src="https://user-images.githubusercontent.com/12522275/108745425-e6049d80-74ef-11eb-9c5e-97470dfb176c.png">

Reply:
<img width="417" alt="Screen Shot 2021-02-22 at 11 38 32 AM" src="https://user-images.githubusercontent.com/42451602/108753856-7ea41900-7502-11eb-901b-510b15b2b3c1.png">
I think we should leave as `small` for now the elevation in grommet is harsher on the bottom for the regular box not sure if that's what we want here doesn't look right

Reply:
Yes agreed, let's leave as small for now.